### DISCUSSION TITLE
Can the vulkano_shaders crate be used?

### DISCUSSION THREAD

**Discussion by coolcatcoder** [cross-rs/cross#1417] [Q&A] [answered]
Having some difficulty with vulkano_shaders due to it requiring the vulkan sdk. Usually this just requires setting some environment variables, so the shader compiler can find it, but with cross I would have no idea how to do this. Is this possible?

**Comment by Emilgardis** [comment, marked as answer]
You'd have to make sure whatever is needed is included in the image cross ends up calling cargo in. This probably means a custom image, see https://github.com/cross-rs/cross/wiki/FAQ#custom-images
